test/test_estimators/test_estimators.py

```diff
@@ -378,6 +378,74 @@ def test_cv_results(tmp_dir):
     assert hasattr(cls, "classes_")
 
 
+def test_cv_results_multi_objective(tmp_dir):
```

Comment:
[Q]
This test seems to be testing two different things, namely, `cv_results_` and some `sklearn` compliance. I'm okay with it as we can make the Estimators as real runs soon but just wondering if it's intentional. I would guess it's to save on compute time

Reply:
I mostly copied the above test which already did this. Using real runs via a fixture and splitting this into two tests would be of course better.